fixtures/events.py

```diff
@@ -8,23 +8,22 @@
 register_event fixture accepts one or several expected events.
 
 simple example:
-    from utils.events import EventBuilder
-    event = EventBuilder().new_event(target_type = 'VmOrTemplate',
-                                     target_name = vm_crud.name,
-                                     event_type = 'vm_create')
+    event = appliance.event_listener().new_event(target_type = 'VmOrTemplate',
+                                                 target_name = vm_crud.name,
+                                                 event_type = 'vm_create')
     register_event(event)
+
 more complex example:
 
-    from utils.events import EventBuilder
-    builder = EventBuilder()
+    listener = appliance.event_listener()
```

Comment:
**Optional**
Consider including a list/dict/set of listeners in the complex example